sdk/storage/storage-blob/src/utils/constants.ts

```diff
@@ -50,3 +50,8 @@ export const HeaderConstants = {
 
 export const ETagNone = "";
 export const ETagAny = "*";
+
+export const REQUIRE_DFS_ENDPOINT_FLAG = "REQUIRE_DFS_ENDPOINT_FLAG";
+
+export const BLOB_SERVICE_PUBLIC_CLOUD_HOST = "blob.core.windows.net";
```

Comment:
You know there could be possiblity that customer are using customized host, in this case replace would fail to work?

Reply:
Yes. I can export the `const ENDPOINT_REPLACEMENTS = [[BLOB_SERVICE_PUBLIC_CLOUD_HOST, DFS_SERVICE_PUBLIC_CLOUD_HOST]];` variable. So developers could inject their own replacements.

---
In reply to: [318954666](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r318954666) [](ancestors = 318954666)